<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>789</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NeuroDAQ Analysis</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="selectionTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="dataTab">
         <attribute name="title">
          <string>Data</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTreeView" name="dirTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
            <widget class="h5TreeWidget" name="fileDataTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragOnly</enum>
             </property>
             <property name="defaultDropAction">
              <enum>Qt::CopyAction</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="oneDimAnalysisTab">
         <attribute name="title">
          <string>1D Analysis</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QListView" name="oneDimToolSelect"/>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="avgTool">
              <widget class="QCheckBox" name="checkBox">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>97</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>Max</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="baselineTool"/>
             <widget class="QWidget" name="measureTool"/>
             <widget class="QWidget" name="page_2"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="matplotlibWidget" name="singlePlotWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QTabWidget" name="displayTabWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="plotTab">
        <attribute name="title">
         <string>Plot</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="matplotlibWidget" name="dataPlotsWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tableTab">
        <attribute name="title">
         <string>Table</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="dataTableWidget">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="h5TreeWidget" name="workingDataTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="propsTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionLoadData"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionSaveFileAs"/>
  </widget>
  <action name="actionLoadData">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAddRootGroup">
   <property name="text">
    <string>Add Root Group</string>
   </property>
  </action>
  <action name="actionAddChildGroup">
   <property name="text">
    <string>Add Child Group</string>
   </property>
  </action>
  <action name="actionAddDataset">
   <property name="text">
    <string>Add Dataset</string>
   </property>
  </action>
  <action name="actionRenameTreeItem">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionRemoveTreeItem">
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="actionPlotData">
   <property name="icon">
    <iconset>
     <normaloff>icons/pencil29.png</normaloff>icons/pencil29.png</iconset>
   </property>
   <property name="text">
    <string>plotData</string>
   </property>
   <property name="toolTip">
    <string>Plot Data</string>
   </property>
  </action>
  <action name="actionShowCursors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/push7.png</normaloff>icons/push7.png</iconset>
   </property>
   <property name="text">
    <string>showCursors</string>
   </property>
   <property name="toolTip">
    <string>Show Cursors</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>icons/home107.png</normaloff>icons/home107.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionBaseline">
   <property name="text">
    <string>Baseline</string>
   </property>
   <property name="toolTip">
    <string>Baseline traces</string>
   </property>
  </action>
  <action name="actionAverage">
   <property name="text">
    <string>Average</string>
   </property>
   <property name="toolTip">
    <string>Make Average</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Stats</string>
   </property>
   <property name="toolTip">
    <string>Measure Statistics</string>
   </property>
  </action>
  <action name="actionShowInTable">
   <property name="text">
    <string>Show in Table</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidgetFile</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>h5TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>h5TreeWidgetFile</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
